config.js

```diff
@@ -795,6 +795,9 @@ config.NSFS_GLACIER_MIGRATE_INTERVAL = 15 * 60 * 1000;
 // of `manage_nsfs glacier restore`
 config.NSFS_GLACIER_RESTORE_INTERVAL = 15 * 60 * 1000;
 
+// enable/disable unsorted listing application level
+config.ENABLE_NSFS_UNSORTED_OBJECTS_LIST = false; 
```

Comment:
I would use "NSFS_LIST_OBJECTS_V2_UNSORTED_" as prefix and add "ENABLED" as suffix to the config name, so we can also add other suffixes for options of this feature -
```suggestion
config.NSFS_LIST_OBJECTS_V2_UNSORTED_ENABLED = false; 
```